wifiphisher/pywifiphisher.py

```diff
@@ -136,6 +142,14 @@ def check_args(args):
             W +
             '] Pre-shared key must be between 8 and 63 printable characters.')
 
+    if args.handshake_capture and not os.path.isfile(
+            args.handshake_capture):
+        sys.exit('[' +
+                 R +
+                 '-' +
+                 W +
+                 '] handshake capture does not exist.')
+
```

Comment:
Correct, it should only check if the fourway-handshake exists in the pcap at this point.